addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -42,7 +42,11 @@ def gather_image(
     mime_type = __gather_mime_type(blender_shader_sockets, image_data, export_settings)
     name = __gather_name(image_data, export_settings)
 
-    uri = __gather_uri(image_data, mime_type, name, export_settings)
+    if image_data.original is None:
+        uri = __gather_uri(image_data, mime_type, name, export_settings)
+    else:
+        uri = image_data.original.filepath #TODO make it relative to gltf file (currently is filepath of image in blender)
```

Comment:
There's [code](https://github.com/KhronosGroup/glTF-Blender-IO/blob/423685f18f55d352e10995ac989fc62b422c4d11/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py#L250-L267) in #453 for getting the relative path to an image you can use.